### PR TITLE
docs(readme): added readme

### PR DESCRIPTION
**Pull request by simonecorsi**
I've formatted the README and added relevant info!

### CODE REVIEW

**Comment by antoniomuso** [comment]
For the rest, LGTM! Thank you!

File: README.md [hunk 1]
```diff
@@ -1,37 +1,121 @@
...
+### Uncompress
 
-## Performance
+```js
+const { uncompress } = require('lz4-napi');
 
-### Hardware
+// if you support top-level await
+const compressedBuffer = await getFromSomeStorage();
+const uncompressedBuffer = await compress(compressedBuffer)
```
[the remark this answers is not in the document]
Comment (simonecorsi) [reply README.md]: Done

File: README.md [hunk 2]
```diff
@@ -1,37 +1,121 @@
...
+  * [Compress](#compress)
+  * [Uncompress](#uncompress)
+- [APIs](#apis)
+  * [Promises](#promises)
+    + [`compress`](#compress)
+    + [`uncompress`](#uncompress)
+  * [Sync](#sync)
+    + [`compressSync`](#compresssync)
+    + [`uncompressSync`](#uncompresssync)
+- [Benchmarks](#benchmarks)
```
[the remark this answers is not in the document]
Comment (simonecorsi) [reply README.md:17]: Removed duplicated title, moved benchmark a step down

**Review by antoniomuso — APPROVED** [review]

**Comment by antoniomuso** [comment]
LGTM! Thank you!

**Pull request merged**